scrape.py

```diff
@@ -427,6 +427,7 @@ def add_pretty_names(instances):
         'hi1': 'HI1. High I/O',
         'hs1': 'High Storage',
         'i3': 'I3 High I/O',
+        'i3en': 'I3 High I/O',
```

Comment:
What's the difference between these instance types? Maybe we should explain a bit more here

Reply:
AWS groups both i3 and i3en instance types under "Storage Optimized". https://aws.amazon.com/ec2/instance-types/
The blog post announcing them (https://aws.amazon.com/blogs/aws/new-the-next-generation-i3en-of-i-o-optimized-ec2-instances/) basically says "more denser storage & I/O". What is higher than "High I/O"?